### ISSUE TITLE
Request: Add Part Categories

### ISSUE THREAD

**Issue by Heathcliff01** [HarshdeepKahlon/buildcores-public#24]
Will there be functionality added for custom part categories such as liquid cooling, as well as other peripherals?

**Comment by HarshdeepKahlon** [comment]
Yes, I am planning on adding that in the future. If the category isn't available in the app, you will be able to add it yourself.
